{
  "uid" : "94a6729a294dd85c",
  "name" : "Verify if links in the section 2 lead to the correct pages after clicking",
  "fullName" : "tests.contacts_page_test.TestContactsPageLinks#test_cp_03_02_verify_links_lead_to_the_correct_pages",
  "historyId" : "97fb5c32cce233ffcb41fecd0d4b116e",
  "time" : {
    "start" : 1728784111549,
    "stop" : 1728784111549,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x564a097eb02a <unknown>\n#1 0x564a094d15e0 <unknown>\n#2 0x564a094c97eb <unknown>\n#3 0x564a094b9a09 <unknown>\n#4 0x564a094bb744 <unknown>\n#5 0x564a094b9cbd <unknown>\n#6 0x564a094b952f <unknown>\n#7 0x564a094b9422 <unknown>\n#8 0x564a094b7493 <unknown>\n#9 0x564a094b7ada <unknown>\n#10 0x564a094d3f41 <unknown>\n#11 0x564a095658c5 <unknown>\n#12 0x564a09545b22 <unknown>\n#13 0x564a09564d7d <unknown>\n#14 0x564a095458c3 <unknown>\n#15 0x564a095136b3 <unknown>\n#16 0x564a0951468e <unknown>\n#17 0x564a097b5a2b <unknown>\n#18 0x564a097b99b1 <unknown>\n#19 0x564a097a2225 <unknown>\n#20 0x564a097ba532 <unknown>\n#21 0x564a0978738f <unknown>\n#22 0x564a097d9f28 <unknown>\n#23 0x564a097da0f3 <unknown>\n#24 0x564a097e9e7c <unknown>\n#25 0x7f2fbb494ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"d0215a23e077ee2b2327e473af4a3dfe\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f744800>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x564a097da0f3 \\\\u003Cunknown>\\\\n#24 0x564a097e9e7c \\\\u003Cunknown>\\\\n#25 0x7f2fbb494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x564a097eb02a <unknown>\nE       #1 0x564a094d15e0 <unknown>\nE       #2 0x564a094c97eb <unknown>\nE       #3 0x564a094b9a09 <unknown>\nE       #4 0x564a094bb744 <unknown>\nE       #5 0x564a094b9cbd <unknown>\nE       #6 0x564a094b952f <unknown>\nE       #7 0x564a094b9422 <unknown>\nE       #8 0x564a094b7493 <unknown>\nE       #9 0x564a094b7ada <unknown>\nE       #10 0x564a094d3f41 <unknown>\nE       #11 0x564a095658c5 <unknown>\nE       #12 0x564a09545b22 <unknown>\nE       #13 0x564a09564d7d <unknown>\nE       #14 0x564a095458c3 <unknown>\nE       #15 0x564a095136b3 <unknown>\nE       #16 0x564a0951468e <unknown>\nE       #17 0x564a097b5a2b <unknown>\nE       #18 0x564a097b99b1 <unknown>\nE       #19 0x564a097a2225 <unknown>\nE       #20 0x564a097ba532 <unknown>\nE       #21 0x564a0978738f <unknown>\nE       #22 0x564a097d9f28 <unknown>\nE       #23 0x564a097da0f3 <unknown>\nE       #24 0x564a097e9e7c <unknown>\nE       #25 0x7f2fbb494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784112320,
      "stop" : 1728784112748,
      "duration" : 428
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x564a097eb02a <unknown>\n#1 0x564a094d15e0 <unknown>\n#2 0x564a094c97eb <unknown>\n#3 0x564a094b9a09 <unknown>\n#4 0x564a094bb744 <unknown>\n#5 0x564a094b9cbd <unknown>\n#6 0x564a094b952f <unknown>\n#7 0x564a094b9422 <unknown>\n#8 0x564a094b7493 <unknown>\n#9 0x564a094b7ada <unknown>\n#10 0x564a094d3f41 <unknown>\n#11 0x564a095658c5 <unknown>\n#12 0x564a09545b22 <unknown>\n#13 0x564a09564d7d <unknown>\n#14 0x564a095458c3 <unknown>\n#15 0x564a095136b3 <unknown>\n#16 0x564a0951468e <unknown>\n#17 0x564a097b5a2b <unknown>\n#18 0x564a097b99b1 <unknown>\n#19 0x564a097a2225 <unknown>\n#20 0x564a097ba532 <unknown>\n#21 0x564a0978738f <unknown>\n#22 0x564a097d9f28 <unknown>\n#23 0x564a097da0f3 <unknown>\n#24 0x564a097e9e7c <unknown>\n#25 0x7f2fbb494ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784112320,
        "stop" : 1728784112747,
        "duration" : 427
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x564a097eb02a <unknown>\n#1 0x564a094d15e0 <unknown>\n#2 0x564a094c97eb <unknown>\n#3 0x564a094b9a09 <unknown>\n#4 0x564a094bb744 <unknown>\n#5 0x564a094b9cbd <unknown>\n#6 0x564a094b952f <unknown>\n#7 0x564a094b9422 <unknown>\n#8 0x564a094b7493 <unknown>\n#9 0x564a094b7ada <unknown>\n#10 0x564a094d3f41 <unknown>\n#11 0x564a095658c5 <unknown>\n#12 0x564a09545b22 <unknown>\n#13 0x564a09564d7d <unknown>\n#14 0x564a095458c3 <unknown>\n#15 0x564a095136b3 <unknown>\n#16 0x564a0951468e <unknown>\n#17 0x564a097b5a2b <unknown>\n#18 0x564a097b99b1 <unknown>\n#19 0x564a097a2225 <unknown>\n#20 0x564a097ba532 <unknown>\n#21 0x564a0978738f <unknown>\n#22 0x564a097d9f28 <unknown>\n#23 0x564a097da0f3 <unknown>\n#24 0x564a097e9e7c <unknown>\n#25 0x7f2fbb494ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"d0215a23e077ee2b2327e473af4a3dfe\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728784111549,
      "stop" : 1728784112320,
      "duration" : 771
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784112782,
      "stop" : 1728784112844,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Contacts Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "contacts_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestContactsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.contacts_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 58,
        "unknown" : 0,
        "total" : 61
      },
      "items" : [ {
        "uid" : "b8e420ceb1690e5e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/b8e420ceb1690e5e",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x556e9ac3702a <unknown>\n#1 0x556e9a91d5e0 <unknown>\n#2 0x556e9a9157eb <unknown>\n#3 0x556e9a905a09 <unknown>\n#4 0x556e9a907744 <unknown>\n#5 0x556e9a905cbd <unknown>\n#6 0x556e9a90552f <unknown>\n#7 0x556e9a905422 <unknown>\n#8 0x556e9a903493 <unknown>\n#9 0x556e9a903ada <unknown>\n#10 0x556e9a91ff41 <unknown>\n#11 0x556e9a9b18c5 <unknown>\n#12 0x556e9a991b22 <unknown>\n#13 0x556e9a9b0d7d <unknown>\n#14 0x556e9a9918c3 <unknown>\n#15 0x556e9a95f6b3 <unknown>\n#16 0x556e9a96068e <unknown>\n#17 0x556e9ac01a2b <unknown>\n#18 0x556e9ac059b1 <unknown>\n#19 0x556e9abee225 <unknown>\n#20 0x556e9ac06532 <unknown>\n#21 0x556e9abd338f <unknown>\n#22 0x556e9ac25f28 <unknown>\n#23 0x556e9ac260f3 <unknown>\n#24 0x556e9ac35e7c <unknown>\n#25 0x7f05b0c94ac3 <unknown>",
        "time" : {
          "start" : 1728697225504,
          "stop" : 1728697225504,
          "duration" : 0
        }
      }, {
        "uid" : "21a737932f2a05e4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/21a737932f2a05e4",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5618f193c02a <unknown>\n#1 0x5618f16225e0 <unknown>\n#2 0x5618f161a7eb <unknown>\n#3 0x5618f160aa09 <unknown>\n#4 0x5618f160c744 <unknown>\n#5 0x5618f160acbd <unknown>\n#6 0x5618f160a52f <unknown>\n#7 0x5618f160a422 <unknown>\n#8 0x5618f1608493 <unknown>\n#9 0x5618f1608ada <unknown>\n#10 0x5618f1624f41 <unknown>\n#11 0x5618f16b68c5 <unknown>\n#12 0x5618f1696b22 <unknown>\n#13 0x5618f16b5d7d <unknown>\n#14 0x5618f16968c3 <unknown>\n#15 0x5618f16646b3 <unknown>\n#16 0x5618f166568e <unknown>\n#17 0x5618f1906a2b <unknown>\n#18 0x5618f190a9b1 <unknown>\n#19 0x5618f18f3225 <unknown>\n#20 0x5618f190b532 <unknown>\n#21 0x5618f18d838f <unknown>\n#22 0x5618f192af28 <unknown>\n#23 0x5618f192b0f3 <unknown>\n#24 0x5618f193ae7c <unknown>\n#25 0x7f70e2e94ac3 <unknown>",
        "time" : {
          "start" : 1728610916852,
          "stop" : 1728610916852,
          "duration" : 0
        }
      }, {
        "uid" : "f185c843b06c7fc0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/f185c843b06c7fc0",
        "status" : "passed",
        "time" : {
          "start" : 1728524584968,
          "stop" : 1728524586328,
          "duration" : 1360
        }
      }, {
        "uid" : "2c8e641002700561",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/2c8e641002700561",
        "status" : "passed",
        "time" : {
          "start" : 1728438157221,
          "stop" : 1728438158406,
          "duration" : 1185
        }
      }, {
        "uid" : "bb6b865a84a8361e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/bb6b865a84a8361e",
        "status" : "passed",
        "time" : {
          "start" : 1728351815100,
          "stop" : 1728351816380,
          "duration" : 1280
        }
      }, {
        "uid" : "176f082ba0e010cf",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/176f082ba0e010cf",
        "status" : "passed",
        "time" : {
          "start" : 1728265599972,
          "stop" : 1728265601780,
          "duration" : 1808
        }
      }, {
        "uid" : "4bc32c25bb52b874",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/4bc32c25bb52b874",
        "status" : "passed",
        "time" : {
          "start" : 1728179372972,
          "stop" : 1728179374379,
          "duration" : 1407
        }
      }, {
        "uid" : "c243258ef6ed50c7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/c243258ef6ed50c7",
        "status" : "passed",
        "time" : {
          "start" : 1728092535727,
          "stop" : 1728092537661,
          "duration" : 1934
        }
      }, {
        "uid" : "61d32a07cdb3b123",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/61d32a07cdb3b123",
        "status" : "passed",
        "time" : {
          "start" : 1728006241221,
          "stop" : 1728006243328,
          "duration" : 2107
        }
      }, {
        "uid" : "6a166defd34b87f0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/6a166defd34b87f0",
        "status" : "passed",
        "time" : {
          "start" : 1727919829248,
          "stop" : 1727919831176,
          "duration" : 1928
        }
      }, {
        "uid" : "72e8e548851ea93",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/72e8e548851ea93",
        "status" : "passed",
        "time" : {
          "start" : 1727833381519,
          "stop" : 1727833382821,
          "duration" : 1302
        }
      }, {
        "uid" : "c94a45c8eaee817f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/c94a45c8eaee817f",
        "status" : "passed",
        "time" : {
          "start" : 1727747517794,
          "stop" : 1727747519049,
          "duration" : 1255
        }
      }, {
        "uid" : "5b70d0bc39c80af0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/5b70d0bc39c80af0",
        "status" : "passed",
        "time" : {
          "start" : 1727660799006,
          "stop" : 1727660800339,
          "duration" : 1333
        }
      }, {
        "uid" : "d65c35700a227233",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/d65c35700a227233",
        "status" : "passed",
        "time" : {
          "start" : 1727574601394,
          "stop" : 1727574602644,
          "duration" : 1250
        }
      }, {
        "uid" : "ed72fc1d415618fa",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/ed72fc1d415618fa",
        "status" : "passed",
        "time" : {
          "start" : 1727487853275,
          "stop" : 1727487854572,
          "duration" : 1297
        }
      }, {
        "uid" : "194d6599b4b268bd",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/194d6599b4b268bd",
        "status" : "passed",
        "time" : {
          "start" : 1727401423263,
          "stop" : 1727401425098,
          "duration" : 1835
        }
      }, {
        "uid" : "5208d7cb35788c48",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/5208d7cb35788c48",
        "status" : "passed",
        "time" : {
          "start" : 1727314970378,
          "stop" : 1727314972312,
          "duration" : 1934
        }
      }, {
        "uid" : "6611cf23dd0faf91",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/6611cf23dd0faf91",
        "status" : "passed",
        "time" : {
          "start" : 1727228663276,
          "stop" : 1727228664548,
          "duration" : 1272
        }
      }, {
        "uid" : "ab30f5e3bd324ce3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/ab30f5e3bd324ce3",
        "status" : "passed",
        "time" : {
          "start" : 1727142224735,
          "stop" : 1727142226143,
          "duration" : 1408
        }
      }, {
        "uid" : "74e7ebd40b1fe669",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/74e7ebd40b1fe669",
        "status" : "passed",
        "time" : {
          "start" : 1727055779690,
          "stop" : 1727055780827,
          "duration" : 1137
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "94a6729a294dd85c.json",
  "parameterValues" : [ ]
}